test/integration/cannot_view_another_users_order_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.